pkg/executive/db_limiter.go

```diff
@@ -206,7 +206,7 @@ func (l *dbLimiter) refreshWriterLimits(ctx context.Context) error {
 		if err != nil {
 			return errors.Wrap(err, "adjust found rate limit")
 		}
-		events.Debug("adjusted %v limit from %v/%v to %v/%v", writerName, maxRowsPerMinute, time.Minute, adjustedRate, l.defaultWriterLimit.Period)
+		log.EventDebug("adjusted %v limit from %v/%v to %v/%v", writerName, maxRowsPerMinute, time.Minute, adjustedRate, l.defaultWriterLimit.Period)
```

Comment:
Why are some `events.Debug` changed to `log.Debug` and some `log.EventDebug`?

Reply:
Lines 117 and 209 show an example of this. The source call was `events.Debug` and on line 117 `log.Debug` was used and on line 209 `log.EventDebug` was used.